tfjs-core/src/platforms/platform.ts

```diff
@@ -46,4 +46,6 @@ export interface Platform {
   encode(text: string, encoding: string): Uint8Array;
   /** Decode the provided bytes into a string using the provided encoding. */
   decode(bytes: Uint8Array, encoding: string): string;
+
+  setTimeoutCustom?(functionRef: Function, delay: number): void;
```

Comment:
I changed the parameter names to be aligned with setTimeout definition in browser (https://developer.mozilla.org/en-US/docs/Web/API/setTimeout).